tests/Pay/Adapter/StripeTest.php

```diff
@@ -4,14 +4,15 @@
 
 use PHPUnit\Framework\TestCase;
 use Utopia\Pay\Adapter\Stripe;
+use Utopia\Pay\Exception;
 
 class StripeTest extends TestCase
 {
     private Stripe $stripe;
 
     protected function setUp(): void
     {
-        $secretKey = getenv('STRIPE_SECRET') ? getenv('STRIPE_SECRET') : '';
+        $secretKey = getenv('STRIPE_SECRET') ? getenv('STRIPE_SECRET') : 'sk_test_51LT5nsGYD1ySxNCydTZfiu3E8h6BAb7GWvVGUWOuqTBEWZMkffxAeP5N8xuENQgLs2nmWNJBpFT8lLrwGIEsLvK800rfxozxYD';
```

Comment:
Please revert this

Reply:
Ah yes, this is test env key, but I'll revert this.